packages/landingpage-new/pages/home/component-sample/component-sample.module.scss

```diff
@@ -0,0 +1,84 @@
+@use '../../../../elements/src/components/base/typography';
```

Comment:
You can not use these styles because they are defined in an separate packages and are currently not exported. Until we've made them available, we have to defined these styles ourselves. I would suggest creating a `typography.scss` in `styles/` and write some mixins. E.g.:

```scss
mixin light-font() {
		font-weight: ...;
		font-size: ...;
} 
```

This way, we can easily swap these out later.